react/src/components/admin/layout/AdminHeader.jsx

```
@@ -0,0 +1,62 @@
+                    ) : (
+                        <span className={styles['profile-letter']}>
+                            {user?.name?.charAt(0).toUpperCase()}
+                        </span>
+                    )}
+                </button>
+                {isMenuOpen && (
+                    <div className={styles['dropdown-menu']}>
+                        <Link to="/profile" onClick={() => setIsMenuOpen(false)}>
+                            {t('nav.profile')}
+                        </Link>
+                        <button onClick={handleLogout}>{t('nav.logout')}</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)